strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/io/RepositoryFileSystemProvider.java

```diff
@@ -420,15 +422,15 @@ public InputStream newInputStream(Path path,
                                       OpenOption... options)
             throws IOException
     {
-        return super.newInputStream(unwrap(path), options);
+        return new BufferedInputStream(super.newInputStream(unwrap(path), options));
     }
 
     @Override
     public OutputStream newOutputStream(Path path,
                                         OpenOption... options)
             throws IOException
     {
-        return super.newOutputStream(unwrap(path), options);
+        return new BufferedOutputStream(super.newOutputStream(unwrap(path), options));
```

Comment:
this will probably cause Buffered streams of Buffered streams

Reply:
I've been thinking and thinking... when would you get a buffered stream of a buffered stream? Could you give me an example here?

Reply:
for example with [RepositoryInoutStream](https://github.com/strongbox/strongbox/pull/672/files/8b78495a92cf0297750a476371dfaf59e927c982#diff-9e6c87aefdc23130851930725f382d13R15) which is BufferedInputStream now and it wraps an InputStream from `RepositoryFileSystemProvider`

Reply:
`RepositoryLayoutFileSystemProvider` extends `RepositoryFileSystemProvider`, correct, thanks :) I'll rollback this extra wrapper.

Reply:
Thank you for the clarification, @sbespalov!